xbmc/music/Song.cpp

```diff
@@ -221,6 +222,7 @@ void CSong::Serialize(CVariant& value) const
   value["duration"] = iDuration;
   value["track"] = iTrack;
   value["year"] = iYear;
+  //value['disctitle'] = strDiscSubtitle;
```

Comment:
Commented out to get it to compile (on Windows) - investiate when JSON implemented